mp4/file.go

```diff
@@ -105,47 +116,34 @@ func (f *File) AddMediaSegment(m *MediaSegment) {
 }
 
 // DecodeFile - parse and decode a file from reader r
-func DecodeFile(r io.Reader) (*File, error) {
+func DecodeFile(r io.Reader, options ...Option) (*File, error) {
```

Comment:
The options must be explained in the documentation of the new parameters. We also need example/test code for them.

Reply:
added a couple of test cases for the DecodeFile func